scratch/evaluate_gaming_traffic.cc

```diff
@@ -25,30 +25,85 @@
 #include "ns3/internet-module.h"
 #include "ns3/point-to-point-module.h"
 #include "ns3/applications-module.h"
-#include "ns3/packet-sink.h"
-#include "ns3/random-variable-stream.h"
 
 using namespace ns3;
 using namespace std;
 
 NS_LOG_COMPONENT_DEFINE ("EvaluateGamingTraffic");
 
+Time delayAccumulator;
+Ptr<GamingStreamingServer> gamingServer;  /* Pointer to the gaming server application */
+Ptr<PacketSink> packetSink;    /* Pointer to the packet sink application */
+uint64_t lastTotalRx = 0;      /* The value of the last total received bytes */
+Time lastPacketTime = Seconds (-1.0);
+
+
+void
+GeneratedPacketsStats (Ptr<const Packet> packet)
+{
+  Time currentPacketTime = Simulator::Now ();
+  if (lastPacketTime.IsStrictlyNegative ())
+    {
+      lastPacketTime = currentPacketTime;
+    }
+  Time packetsInterArrivalTime = currentPacketTime - lastPacketTime;
+  lastPacketTime = currentPacketTime;
+  std::cout << "" << packet->GetSize ()
```

Comment:
if possible, avoid using `std::cout` in favor of `NS_LOG_UNCOND`. In this case, though, it would be better to write the output to a file